pkgs/applications/misc/kickoff/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, fontconfig
+, pkg-config
+, wayland
+, libxkbcommon
+, makeWrapper
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "kickoff";
+  version = "v0.7.0";
+
+  src = fetchFromGitHub {
+    owner = "j0ru";
+    repo = pname;
+    rev = version;
+    hash = "sha256-AolJXFolMEwoK3AtC93naphZetytzRl1yI10SP9Rnzo=";
+  };
+
+  cargoHash = "sha256-OEFCR/2zSVZhZqAp6n48UyIwplRXxKb9HENsVaLIKkM=";
+
+  libPath = lib.makeLibraryPath [
+    wayland
+    libxkbcommon
+  ];
+
+  buildInputs = [ fontconfig ];
+  nativeBuildInputs = [ makeWrapper pkg-config ];
+
+  postInstall = ''
+    wrapProgram "$out/bin/kickoff" --prefix LD_LIBRARY_PATH : "${libPath}"
+  '';
+
+  meta = with lib; {
+    description = "Minimalistic program launcher";
+    homepage = "https://github.com/j0ru/kickoff";
+    license = licenses.gpl3Plus;
+    mainProgram = "kickoff";
```

Comment:
```suggestion
```
This is not necessary when only one binary is. Sorry for the many time's suggestions.

Reply:
Ok I removed it. No worries. Thanks for taking the time!